xbmc/video/VideoThumbLoader.cpp

```diff
@@ -436,7 +436,7 @@ bool CVideoThumbLoader::FillLibraryArt(CFileItem &item)
     m_videoDatabase->Open();
     if (m_videoDatabase->GetArtForItem(tag.m_iDbId, tag.m_type, artwork))
       SetArt(item, artwork);
-    else if (tag.m_type == MediaTypeArtist)
+    else if (tag.m_type == "actor" && !tag.m_artist.empty())
```

Comment:
thanks. best we introduce a new kind of type for that stuff instead of violate `m_type`.